src/extension.ts

```diff
@@ -36,6 +36,13 @@ export class EdgeConfigurationProvider implements vscode.DebugConfigurationProvi
      * Try to add all missing attributes to the debug configuration being launched.
      */
     async resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, config: vscode.DebugConfiguration, token?: vscode.CancellationToken): Promise<vscode.DebugConfiguration> {
+        if (!isEdgeDebuggingSupported()) {
+            const errorMessage = localize('edge.debug.error.versionNotSupported', "Edge debugging is not supported for your version of Windows. Please update to the latest version.");
+            return vscode.window.showErrorMessage(errorMessage).then(_ => {
```

Comment:
Is the caller of this method going to show an error when we return undefined? It it possible we are going to get duplicated error messages or something like that if we both show an error message and return a failure?

Reply:
This was guidance that we got from the vscode team on how to implement this check. In my testing I only ever get one error message.